Packs/Stairwell/Integrations/Inception/Inception.py

```diff
@@ -72,16 +85,15 @@ def file_enrichment_command(client, file_hash):
                 yara_string = (', '.join([str(x) for x in yara_rules]))
                 md += f'Matching YARA Intel: {yara_string}\n'
 
-            # Create readable output if AV results exist
-            if responseJson['data']['attributes']['last_analysis_results']:
-                avResults = responseJson['data']['attributes']['last_analysis_results']
-                md += '### AV Scanning Results\n'
-                md += 'Engine Name|Result\n'
-                md += '---|---\n'
-                for indAv in avResults:
-                    engine_name = avResults[indAv]['engine_name']
-                    result = avResults[indAv]['result']
-                    md += f'{engine_name}|{result}\n'
+            # Show Mal-Eval verdict
+            mal_eval_output = responseJson['data']['attributes']['mal_eval_result']
+            if mal_eval_output:
+                if mal_eval_output['label'] != '':
```

Comment:
n python empty strings are "falsy" [(see here)](https://docs.python.org/3/library/stdtypes.html#truth-value-testing)

if mal_eval_output['label'] returns True then it is not an empty string by definition.
```suggestion
                if mal_eval_output['label']:
```

Reply:
Fixed!